experimental/normalizer/Cargo.toml

```
@@ -0,0 +1,61 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_normalizer"
+description = "API for normalizing text into Unicode Normalization Forms"
+version = "0.5.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "LICENSE"
+categories = ["internationalization"]
+# Keep this in sync with other crates unless there are exceptions
+include = [
+    "src/**/*",
+    "examples/**/*",
+    "benches/**/*",
+    "tests/**/*",
+    "Cargo.toml",
+    "LICENSE",
+    "README.md"
+]
+
+[package.metadata.cargo-all-features]
+skip_optional_dependencies = true
+# Bench feature gets tested separately and is only relevant for CI
+denylist = ["bench"]
+
+[package.metadata.docs.rs]
+all-features = true
+
+[dependencies]
+smallvec = "1.6"
+displaydoc = { version = "0.2.3", default-features = false }
+#num_enum = { version = "0.5", default_features = false }
+icu_codepointtrie = { version = "0.3", path = "../../utils/codepointtrie", features = ["serde"] }
+icu_provider = { version = "0.5", path = "../../provider/core", features = ["macros"] }
+serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
+zerovec = { version = "0.6", path = "../../utils/zerovec", features = ["serde"] }
+icu_uniset = { version = "0.4", path = "../../utils/uniset", features = ["serde"] }
+icu_properties = { version = "0.5", path = "../../components/properties" }
+
+[dev-dependencies]
+#criterion = "0.3"
+#icu = { path = "../icu", default-features = false }
+#icu_benchmark_macros = { version = "0.3", path = "../../tools/benchmark/macros" }
+#icu_provider = { version = "0.3", path = "../../provider/core" }
+icu_testdata = { version = "0.5", path = "../../provider/testdata", features = ["static"] }
+#serde = { version = "1.0", features = ["derive"] }
+#serde_json = "1.0"
+#bincode = "1.3"
+
+[lib]
+path = "src/lib.rs"
+bench = false  # This option is required for Benchmark CI
+
+[features]
+default = ["provider_serde"]
+provider_serde = ["serde"]
```

```suggestion
default = []
serde = ["dep:serde", "icu_codepointtrie/serde", "zerovec/serde", "icu_uniset/serde"]
datagen = ["serde", "icu_codepointtrie/serde_serialize", "zerovec/serde_serialize", "icu_unitset/serde_serialize"]
```
and remove the `serde` feature on dependencies above.

---

Done. Thanks.